src/main/kotlin/dk/cachet/carp/webservices/data/domain/CawsMutableDataStreamBatch.kt

```diff
@@ -0,0 +1,94 @@
+package dk.cachet.carp.webservices.data.domain
```

Comment:
It would make more sense for me to keep this file under service, or service/impl, since it is not a domain entity.